### DISCUSSION TITLE
Unibody Picocomputer build

### DISCUSSION THREAD

**Discussion by grakoczy** [picocomputer/community#123] [Q&A]
Hi,

I've just finished building unibody version of the picocomputer. 
It's using some dirrt cheap usb keyboard with blue switches so they sound "retro" clicky. It also has RGB light if I feel fancy. I had to replace keycaps with Keychron retro set I had laying around. 
USB hub I've found in the drawer has another input for keyboard on another side so no extra soldering was required.
The case was printed with PLA in two pieces as it's too big for my printer and then painted. The badge is printed on resin printer and then hand painted. 
It's still a prototype but I'll try use it a little bit before I start making changes.
Here are some photos and render from Fusion 360 of the design.

![Photo 2)](https://github.com/user-attachments/assets/76169390-959b-40dc-9597-fd07fdd103b2)
![Photo 3](https://github.com/user-attachments/assets/6315810a-31b9-4f48-849e-ab75195631c2)
![Photo 1)](https://github.com/user-attachments/assets/d4b21a5b-875c-45af-81d8-7037e7883def)
<img width="959" alt="picocomputer_render" src="https://github.com/user-attachments/assets/1810a424-ce6b-4cc9-b0da-a09b95d5c40f">

**Comment by vruumllc** [comment]
Bravo!!! Gorgeous.

**Comment by GlennFR** [comment]
Very smart!  What sort of kbd base did you use, I couldn't find anything similar on A-xpress ?
Thanks for sharing

**Comment by rumbledethumps** [comment]
That's a lot more compact than I would have thought it could be. Very nice.

**Comment by grakoczy** [comment]
Hi,
I've bougth used Cepter Alpha Pro keyboard. It had very ugly keys but I had nice one feom Keychron.

**Comment by WojciechGw** [comment]
Excellent job :)
